cli/sbomcommands.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/common/cliutils"
+	commandsCommon "github.com/jfrog/jfrog-cli-core/v2/common/commands"
+	"github.com/jfrog/jfrog-cli-core/v2/common/spec"
+	pluginsCommon "github.com/jfrog/jfrog-cli-core/v2/plugins/common"
+	"github.com/jfrog/jfrog-cli-core/v2/plugins/components"
+	coreConfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	flags "github.com/jfrog/jfrog-cli-security/cli/docs"
+	enrichDocs "github.com/jfrog/jfrog-cli-security/cli/docs/enrich"
+	"github.com/jfrog/jfrog-cli-security/commands/enrich"
+)
+
+func getSbomCommands() []components.Command {
+	return []components.Command{
+		{
+			Name:        "enrich",
+			Aliases:     []string{"se"},
+			Flags:       flags.GetCommandFlags(flags.Enrich),
+			Description: enrichDocs.GetDescription(),
+			Arguments:   enrichDocs.GetArguments(),
+			Action:      EnrichCmd,
+		},
+	}
+}
+
+func createServerDetails(c *components.Context) (*coreConfig.ServerDetails, error) {
+	return pluginsCommon.CreateServerDetailsWithConfigOffer(c, true, cliutils.Sbom)
+}
+
+func EnrichCmd(c *components.Context) error {
+	if len(c.Arguments) == 0 {
+		return pluginsCommon.PrintHelpAndReturnError("providing a <source pattern> argument is mandatory", c)
+	}
+	serverDetails, err := createServerDetails(c)
+	if err != nil {
+		return err
+	}
+	err = validateXrayContext(c, serverDetails)
+	if err != nil {
+		return err
+	}
+	specFile := createDefaultScanSpec(c, addTrailingSlashToRepoPathIfNeeded(c))
```

Comment:
```suggestion
    var specFile *spec.SpecFiles
	if c.IsFlagSet(flags.SpecFlag) && len(c.GetStringFlagValue(flags.SpecFlag)) > 0 {
		specFile, err = pluginsCommon.GetFileSystemSpec(c)
		if err != nil {
			return err
		}
	} else {
		specFile = createDefaultScanSpec(c, addTrailingSlashToRepoPathIfNeeded(c))
	}
```
There are more flags and options that you can support by using and creating the scan spec,
check out the flags that are used in `createDefaultScanSpec`, if you want to use them, you need to add it.
also, if not default, you need to load the file that is in the SpecFile value.

Reply:
not sure i understand